packages/nuxt/src/runtime/server/clerkClient.ts

```diff
@@ -0,0 +1,32 @@
+import { createClerkClient } from '@clerk/backend';
+import { isTruthy } from '@clerk/shared/underscore';
+import type { H3Event } from 'h3';
+
+// @ts-expect-error: Nitro import. Handled by Nuxt.
+// eslint-disable-next-line import/no-unresolved
+import { useRuntimeConfig } from '#imports';
+
+export function clerkClient(event: H3Event) {
+  const runtimeConfig = useRuntimeConfig(event);
```

Comment:
This is the server-side [`useRuntimeConfig`](https://nuxt.com/docs/api/composables/use-runtime-config) and requires an `H3Event` to be passed in so that it can properly parse environment variables in different runtimes